src/utils/UrlUtils.ts

```diff
@@ -101,7 +101,7 @@ export class UrlUtils {
       const mapped = paired.map(pair => {
         const [key, value] = pair;
 
-        if (!value || !key) {
+        if (UrlUtils.valueShouldBeRemovedFromQueryString(value) || UrlUtils.valueShouldBeRemovedFromQueryString(key)) {
```

Comment:
I think it would be better if the name of the utility function remains agnostic to what should be done. Something along the lines of `isValidQueryStringValue()`? In this particular instance, the result of the call will be to remove it because we are inside the `normalizeQueryString` method, but if we were to use the method in another place, maybe the action would be to skip parsing the value, rather than remove it.